lib/template.html

```diff
@@ -65,6 +65,10 @@
   {% assign seo_author_twitter = seo_author_twitter | replace:"@","" %}
 {% endif %}
 
+{% if page.date_modified or page.last_modified_at or page.date %}
+  {% assign seo_date_modified = page.seo.date_modified | default: page.last_modified_at | default: page.date %}
```

Comment:
I don't know if we want to set `seo_date_modified` at all if all we have is `page.date`. What is the motivation?